packages/pygments.rb

```diff
@@ -22,6 +22,6 @@ class Pygments < Package
   })
 
   def self.install
-    system "pip3 install pygments --prefix #{CREW_PREFIX} --root #{CREW_DEST_DIR}"
+    system "pip3 install pygments --no-warn-script-location --prefix #{CREW_PREFIX} --root #{CREW_DEST_DIR}"
```

Comment:
Remove binaries to replace with new version?

Reply:
Any reason not to get rid of the binaries entirely and just put in 
```
  source_url 'file:///dev/null'
  source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
```
(because what of the source are we really using?)

Reply:
Yeah, no reason to download the source if it isn't being used.  It will just take longer to install without binaries.